buildpacks/java/Makefile

```diff
@@ -157,8 +159,16 @@ buildpacks.java.tests.sources := \
 # Targets
 #########
 
-buildpacks.java.images.local: buildpacks.java.invoker.url=file://$(abspath $(invokers.out_dir)/java/java-function-invoker-$(buildpacks.java.invoker.version).jar)
-buildpacks.java.images.local: buildpacks.java.images.offline
+buildpacks.java.invoker.path = $(abspath $(invokers.out_dir)/java/java-function-invoker-$(buildpacks.java.invoker.version).jar)
+
+buildpacks.java.images.local:
+	$(MAKE) \
```

Comment:
Nested make ensures variables are set _before_ target dependency graph is generated by make.

`...sha.local` var points to invoker's sha file. If the invoker code is updated and hence rebuilt, this will cause the buildpack to pull in the freshest invoker.

Same applies for python later in this PR

Reply:
Shoutout to @andrew-su for the nested make idea 😁